Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -4053,6 +4339,327 @@ def get_papi_edgehostname_creation_status_command(client: Client,
+        value_to_josn:
+
+    Returns:
+        human readable (markdown format), entry context and raw response
+    """
+
+    import json
+
+    body = [
+        {
+            "op": operation,
+            "path": path,
+            "value": json.loads(value) if value_to_json.lower() =="yes" else value
+        }
+    ]
+    
+    raw_response: Dict = client.patch_papi_property_rule(contract_id=contract_id,
+                                                         group_id=group_id,
+                                                         property_id=property_id,
+                                                         property_version=property_version,
+                                                         validate_rules=validate_rules,
+                                                         body=body,
+                                                         )
+
+    title = f'{INTEGRATION_NAME} - Patch papi property rule command'
+    entry_context, human_readable_ec = patch_papi_property_rule_ec(raw_response)
+    context_entry: Dict = {
+        f"{INTEGRATION_CONTEXT_NAME}.PapiProperty": entry_context
+    }
+    human_readable = tableToMarkdown(
+        name=title,
+        t=human_readable_ec,
+        removeNull=True,
+    )
+
+    return human_readable, context_entry, raw_response
+
+
+@logger
+def get_papi_property_rule_command(client: Client,
+                                   contract_id: str,
+                                   property_id: str,
+                                   property_version: int,
+                                   group_id: str,
+                                   validate_rules: str
+                                   ) -> Tuple[str, Dict[str, Any], Union[List, Dict]]:
+    """
+        Get Papi Property default rule
+    Args:
+         contract_id:
+         property_id:
+         property_version:
+         group_id:
+         validateRules:
+
+    Returns:
+        human readable (markdown format), entry context and raw response
+    """
+
+    import json
+
+    raw_response: Dict = client.get_papi_property_rule(contract_id=contract_id,
+                                                       group_id=group_id,
+                                                       property_id=property_id,
+                                                       property_version=property_version,
+                                                       validate_rules=validate_rules,
+                                                       )
+
+    title = f'{INTEGRATION_NAME} - get papi property default rule command'
+    entry_context = json.dumps(raw_response)
+    human_readable_ec = json.dumps(raw_response)
+    context_entry: Dict = {
+        f"{INTEGRATION_CONTEXT_NAME}.PapiProperty.DefaultRule": entry_context
+    }
+    # human_readable = tableToMarkdown(
+    #     name=title,
+    #     t=human_readable_ec,
+    #     removeNull=True,
+    # )
+    human_readable = json.dumps(raw_response)
```

Comment:
?

Reply:
Sure, this part has been updated under get_papi_property_rule_command, please check.